beetsplug/convert.py

```diff
@@ -385,48 +385,51 @@ def copy_album_art(self, album, dest_dir, path_formats, pretend=False):
                 util.copy(album.artpath, dest)
 
     def convert_func(self, lib, opts, args):
-        if not opts.dest:
-            opts.dest = self.config['dest'].get()
-        if not opts.dest:
+        dest = opts.dest or self.config['dest'].get()
+        if not dest:
             raise ui.UserError(u'no convert destination set')
-        opts.dest = util.bytestring_path(opts.dest)
+        dest = util.bytestring_path(dest)
 
-        if not opts.threads:
-            opts.threads = self.config['threads'].get(int)
+        threads = opts.threads or self.config['threads'].get(int)
 
-        if self.config['paths']:
-            path_formats = ui.get_path_formats(self.config['paths'])
-        else:
-            path_formats = ui.get_path_formats()
-
-        if not opts.format:
-            opts.format = self.config['format'].as_str().lower()
+        path_formats = ui.get_path_formats(self.config['paths'] or None)
 
-        pretend = opts.pretend if opts.pretend is not None else \
-            self.config['pretend'].get(bool)
-
-        if not pretend:
-            ui.commands.list_items(lib, ui.decargs(args), opts.album)
+        fmt = opts.format or self.config['format'].as_str().lower()
 
-            if not (opts.yes or ui.input_yn(u"Convert? (Y/n)")):
-                return
+        if opts.pretend is not None:
+            pretend = opts.pretend
+        else:
+            pretend = self.config['pretend'].get(bool)
 
         if opts.album:
             albums = lib.albums(ui.decargs(args))
-            items = (i for a in albums for i in a.items())
-            if self.config['copy_album_art']:
-                for album in albums:
-                    self.copy_album_art(album, opts.dest, path_formats,
-                                        pretend)
+            items = [i for a in albums for i in a.items()]
+            if not pretend:
+                for a in albums:
+                    ui.print_(format(a, u''))
         else:
-            items = iter(lib.items(ui.decargs(args)))
-        convert = [self.convert_item(opts.dest,
+            items = list(lib.items(ui.decargs(args)))
+            if not pretend:
+                for i in items:
+                    ui.print_(format(i, u''))
+
+        if not items:
+            self._log.info(u'No item to convert.')
```

Comment:
I might suggest the wording "Nothing to convert." because saying "items" could be confusing when the user asked to convert albums.